page2hocr.xsl

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<xsl:stylesheet 
+	xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl"
+	xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:p1="http://schema.primaresearch.org/PAGE/gts/pagecontent/2009-03-16"
+	xmlns:p2="http://schema.primaresearch.org/PAGE/gts/pagecontent/2010-01-12"
+	xmlns:p3="http://schema.primaresearch.org/PAGE/gts/pagecontent/2010-03-19"
+	xmlns:p4="http://schema.primaresearch.org/PAGE/gts/pagecontent/2013-07-15"
+	xmlns:p5="http://schema.primaresearch.org/PAGE/gts/pagecontent/2016-07-15"
+	xmlns:p6="http://schema.primaresearch.org/PAGE/gts/pagecontent/2017-07-15"
+	xmlns:p7="http://schema.primaresearch.org/PAGE/gts/pagecontent/2018-07-15"
+	xmlns="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"
+	exclude-result-prefixes="#all"
+	version="2.0">
+
+
+	<xsl:output method="html" encoding="UTF-8" standalone="yes" indent="yes"/>
+
+
+	<xsl:template match="*:PcGts" xmlns="http://www.w3.org/1999/xhtml">
+		<html>
+			<head>
+				<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+				<link rel="stylesheet" href=""/>
+				<title>OCR Output</title>
+				<meta name="description" content="OCR Output via XSLT of pageXML."/>
+			</head>
+			<body>
+				<xsl:element name="div">
+					<xsl:attribute name="class">ocr_page</xsl:attribute>
+					<xsl:attribute name="title">bbox 0 0 <xsl:value-of select="*:Page/@imageWidth"/>
+						<xsl:text> </xsl:text> <xsl:value-of select="*:Page/@imageHeight"/>
+						<xsl:text>; image 'image/</xsl:text> <xsl:value-of
+						select="*:Page/@imageFilename"/> <xsl:text>';</xsl:text> </xsl:attribute>
+					<xsl:for-each select="*:Page/*:TextRegion">
+						<xsl:call-template name="textregion"/>
+					</xsl:for-each>
+					<xsl:for-each select="*:Page/*:ImageRegion">
+						<xsl:call-template name="imageregion"/>
+					</xsl:for-each>
+					<xsl:for-each select="*:Page/*:GraphicRegion">
+						<xsl:call-template name="graphicregion"/>
+						<xsl:if test="child::*:TextRegion">
+							<xsl:for-each select="*:TextRegion">
+								<xsl:call-template name="textregion"/>
+							</xsl:for-each>
+						</xsl:if>
+					</xsl:for-each>
+				</xsl:element>
+			</body>
+		</html>
+	</xsl:template>
+
+	
+	<xsl:template name="imageregion" xmlns="http://www.w3.org/1999/xhtml">
+		<xsl:param name="bbox_poly">
+			<xsl:call-template name="bbox_or_poly"/>
+		</xsl:param>
+		<div class="cr_carea" title="{$bbox_poly}">
```

Comment:
Where does the class name `cr_carea` come from? It caused confusions (see https://github.com/UB-Mannheim/ocr-fileformat/issues/183), and I could not find it in the hOCR specification.